Dockerfile

```diff
@@ -47,4 +47,5 @@ RUN python manage.py collectstatic --noinput && \
 # Launch
 USER appuser
 EXPOSE 8080
+HEALTHCHECK CMD bash -c 'wget -q -o /dev/null -O /dev/null --header "Host: ${ALLOWED_HOSTS%,*}" "http://127.0.0.1:$PORT/healthz/?format=json"'
```

Comment:
Isn't `-q` acting same as `-o /dev/null`?

Also, wouldn't `curl -fso /dev/null -H "Host: ${ALLOWED_HOSTS%,*}` be better?

Reply:
This is about exit status code and I'm using busybox wget which is always available on alpine images.
I had trouble with making curl return error code like I'd like it to do.
It needs special flag for this and it returns other code than I want - masking it with ` || exit 1` would be needed which is not clean.
I'd like to stay with busybox wget.

> Isn't -q acting same as -o /dev/null?

To be honest: I've no idea. You may be right but I would need to check.

Reply:
It's not the same:

```
busybox wget -q http://googlesdfds.com
wget: bad address 'googlesdfds.com'
$ busybox wget -o /dev/null http://googlesdfds.com
<nothing>
```

Reply:
removed `-q`, thanks ;)